creator-node/src/fileManager.js

```diff
@@ -295,17 +297,36 @@ async function saveFileForMultihashToFS (serviceRegistry, logger, multihash, exp
     }
 
     // verify that the contents of the file match the file's cid
+    let fileData = null
+    let fileSize = null
+    try {
+      fileData = (await fs.readFile(expectedStoragePath)).toString().slice(0, 99)
+      fileSize = (await fs.stat(expectedStoragePath)).size
+    } catch (e) {
+      logger.warn('Could not get fs statistics for path', expectedStoragePath)
+    }
     try {
-      decisionTree.push({ stage: 'About to verify the file contents for the CID', vals: multihash, time: Date.now() })
+      decisionTree.push({ stage: 'About to verify the file contents for the CID', vals: multihash, time: Date.now(), fileData, fileSize })
+
+      // if the file is empty, there's likely something wrong so throw an error
+      // there is one case where an empty file could be valid, check for that CID explicitly
+      const fileIsEmpty = fileSize !== null && fileSize === 0
+      if (fileIsEmpty && multihash !== EMPTY_FILE_CID) {
```

Comment:
so if `fileSize === null` ie fs.stat op failed, we just never check right. can you add comment, or make this a bit more readable

Reply:
how about
```
const fileSizeChecked = fileSize !== null
const fileIsEmpty = fileSize === 0
if (fileSizeChecked && fileIsEmpty && multihash !== EMPTY_FILE_CID) {
```